docs/development-workflow.md

```diff
@@ -51,6 +51,18 @@ or limit it as before with:
 DEBUG=calypso:* make run
 ```
 
+## Clearing browser data
+
+Seeing some strange errors that only happen on _your_ browser? You may need to clear your browser's local storage and database. Calypso saves a lot here and if you've changed the way your data is kept (especially in redux state), it can break things in unepxected ways during the transition.
+
+At the browser console line:
+
+```js
+localStorage.clear(); indexedDB.deleteDatabase( 'calypso' );
```

Comment:
Note that this will clear out storage for Chrome/FF. Depending on the browser localForage might choose a different driver. I wonder if we should just expose our localForage instance in dev.

How to clear browser cache is duplicated in https://github.com/Automattic/wp-calypso/blob/master/docs/merge-checklist.md but we do have a lot of docs so  ¯\_(ツ)_/¯